tests/model/operation/transform/attribute.js

```diff
@@ -440,6 +623,21 @@ describe( 'transform', () => {
 				);
 			} );
 
+			it( 'text in different path', () => {
```

Comment:
We will need to work on those names later cause they aren't really helpful (aren't describing exactly what happens). I understand that we don't want to make sentences of description but it would be good to have more meaningful names. BTW. I am not perfect at that either.